project/src/pages/not-found-screen/not-found-screen.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { Fragment } from 'react';
+import { Link } from 'react-router-dom';
+
+function NotFoundScreen(): JSX.Element {
+  return (
+    <Fragment>
```

Comment:
У фрагмента есть более короткая запись, вот такая и импортировать ничего не надо будет
```suggestion
    <>
```